src/main/java/com/jmb/springfactory/model/dto/ConnectedDto.java

```diff
@@ -0,0 +1,16 @@
+package com.jmb.springfactory.model.dto;
+
+import java.util.List;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class ConnectedDto {
+
+    private String username;
+    private String SessionId;
+    private List<String> grantedAuthorities;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "grantedAuthorities" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)